app/main.py

```diff
@@ -1 +1,39 @@
-# write your code here
+import os
+
+
+class PlannedError(Exception):
+    """To create os.mkdir("first/second"), first directory needs to exist."""
```

Comment:
why do you need this custom exception?
if you don't use anywhere?

Reply:
i'll use `FileExistsError` instead of it then